block-storage-attacher/Dockerfile

```diff
@@ -12,6 +12,8 @@ LABEL git-remote-url=${git_remote_url}
 LABEL build-date=${build_date}
 LABEL jenkins-build-number=${jenkins_build_number}
 
+RUN apk add --update
+RUN apk upgrade
```

Comment:
i see these above lines are present in master currently. So check and remove from the PR

Reply:
I think if I remove again merge confilct wil raise. let it be